src/main/java/com/brightinteractive/assetbank/restapi/representations/BatchContentUrlRepr.java

```diff
@@ -0,0 +1,16 @@
+package com.brightinteractive.assetbank.restapi.representations;
+
+import java.util.List;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "batchContentUrls")
```

Comment:
Is the batch prefix on the root element something we've done with other batch responses? If not, I'd probably just go with contentUrls

Reply:
I've changed it to assetContentUrls because contentUrls is already the name of the attribute and we don't want to repeat the XML element names